packages/app-admin-ui/client/pages/Item/index.js

```diff
@@ -413,10 +412,13 @@ const ItemPage = ({ list, itemId, adminPath, getListByKey }) => {
   // try to initialise the fields. They are Suspense capable, so may
   // throw Promises which will be caught by the wrapping <Suspense>
   captureSuspensePromises([
-    ...list.fields
-      .filter(({ isPrimaryKey }) => !isPrimaryKey)
-      .filter(({ maybeAccess }) => !!maybeAccess.update)
-      .map(field => () => field.initFieldView()),
+    // NOTE: We should really filter out non-renderable fields here, but there's
+    // an edgecase where deserialising will include all fields (even
+    // non-renderable ones), so we have to ensure the field views are all loaded
+    // even if they may not be used. This is particularly salient for the
+    // Content field which needs its views loaded to correctly deserialize.
+    // Ideally we'd use getRenderableFields() here.
+    ...list.fields.map(field => () => field.initFieldView()),
```

Comment:
The reason it's important to do this here is later on line 452 we call `deserializeItem` which internally calls `deserialize`, which relies on the fields views having been loaded.